Source/WebCore/editing/VisibleUnits.cpp

```diff
@@ -52,9 +52,6 @@
 
 namespace WebCore {
 
-using namespace HTMLNames;
```

Comment:
Hm.. I think this file references `brTag`, which requires the namespace.

Reply:
@whsieh - Should I undo this line or update call-site to `HTMLNames::brTag`?

Reply:
Either works for me! Up to you.

(It probably still builds right now due to source unification)

Reply:
I will update call-site. :-)

Reply:
@whsieh - done. :-)